docs/server_notices.md

```diff
@@ -61,14 +61,33 @@ displayname and avatar of the Server Notices user.
 Sending notices
 ---------------
 
-As of the current version of synapse, there is no convenient interface for
-sending notices (other than the automated ones sent as part of consent
-tracking).
+The API to send notices is as follows:
```

Comment:
could you stick this under `docs/admin_api`, with a link here to say "there is an API which is documented at [...]"?